src/qtt/algorithms/coulomb.py

```diff
@@ -79,13 +76,14 @@ def fitCoulombPeaks(x_data, y_data, lowvalue=None, verbose=1, fig=None, sampling
     Returns:
         (list): A list with detected peaks.
     """
-    minval = np.percentile(y_data, 5) + 0.1 * (np.percentile(y_data, 95) - np.percentile(y_data, 5))
+    p1, p5, p95, p99 = np.percentile(y_data, [1, 5, 95, 99])
+    minval = p5 + 0.1 * (p95 - p5)
     local_maxima, _ = nonmaxsuppts(y_data, d=int(12 / sampling_rate), minval=minval)
     fit_data = fitPeaks(x_data, y_data, local_maxima, verbose=verbose >= 2, fig=fig)
 
     if lowvalue is None:
-        lowvalue = np.percentile(y_data, 1)
-    highvalue = np.percentile(y_data, 99)
+        lowvalue = p1
```

Comment:
lowvalue parameter should be added to the docstring, explaining this default value when None.